src/lib/CollectStore.js

```diff
@@ -19,90 +23,88 @@ const normalize = (dbObject, doctype) => {
 }
 
 export default class CollectStore {
-  constructor(context, options = {}) {
+  constructor(context, client, options = {}) {
+    this.client = client
     this.listener = null
     this.options = options
 
     this.categories = require('../config/categories')
     this.banksUrl = null
 
+    this.updateUnfinishedJob = this.updateUnfinishedJob.bind(this)
+    this.onAccountCreated = this.onAccountCreated.bind(this)
+    this.onAccountUpdated = this.onAccountUpdated.bind(this)
+    this.onAccountDeleted = this.onAccountDeleted.bind(this)
+    this.onTriggerCreated = this.onTriggerCreated.bind(this)
+    this.onTriggerDeleted = this.onTriggerDeleted.bind(this)
+
     this.initializeRealtime()
   }
 
   async initializeRealtime() {
```

Comment:
I don't think we need `async` here ?

Reply:
Or do we need to await all the subscribtions ?

Reply:
You're right, thanks !